.github/workflows/test.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetesVersion: ["v1.14.10", "v1.22.0"]
+        kubernetesVersion: ["v1.19.0", "v1.22.0", "v1.25.0"]
```

Comment:
I've run into issues with older versions of the `kindest:node` image, the testing for v1.19.0 might pass if you use the latest kindest image `v1.19.16`

Reply:
I'm trying 1.21 instead which is the oldest version EKS supports. You ok with 1.21, 1.22, 1.25?

Reply:
nvm, there is no 1.21.0 kindest/node image so I'm using 1.19.16 anyway.